index.bs

```diff
@@ -1923,6 +1923,34 @@ This section defines an initial set of extensions. These are recommended for imp
 interoperability.
 
 
+## U2F AppId ## {#extension-u2f_appid}
+
+This authentication extension allows [RP]s who have previously registered an
+Authentication using the legacy FIDO U2F JavaScript APIs to communicate with
+Authenticators.  Specifically, this extension allows [RP]s to specify an
+|appId| to overwrite the otherwise computed |rpId|.
+
+: Extension identifier
+:: `webauthn_u2f-appid`
+
+: Client argument
+:: A single UTF-8 encoded string specifying a FIDO U2F |appId|.
+
+: Client processing
+:: AppId validation (as defined by the FIDO U2F specification).  If valid, the
```

Comment:
Need a real reference for this. Please contribute to http://www.specref.org/ or add to the biblio block at the end of the Bikeshed source (it's also in SpecRef format).